test/integration/__tests__/live-api.js

```diff
@@ -487,8 +487,17 @@ describe('API integration', () => {
         // Successful cancel requests get ASSEMBLY_CANCELED even when it
         // completed, so we now request the assembly status to check the
         // *actual* status.
-        const resp2 = await client.getAssembly(id)
-        expect(resp2.ok).toBe('ASSEMBLY_CANCELED')
+
+        const timeout = 30000
```

Comment:
```suggestion
        const timeout = 30_000
```

or
```suggestion
        const timeout = ms('30s')
```

Reply:
For readability

Reply:
cannot do that because we support node>=10. maybe we could bump major version in order to be able to use new features, but i'm not sure if we should do it just for this change
as for `ms`, we could do that but need to add another dependency then

Reply:
Ok I agree, the current state is good then 👍